site/_docs/pages.md

```diff
@@ -77,6 +80,8 @@ might look like:
 |   └── index.html  # => http://example.com/about/
 ├── contact/
 |   └── index.html  # => http://example.com/contact/
+|── other/
+|   └── index.md    # => http://example.com/other.html
```

Comment:
I don't think this URL is correct. I believe it should be `http://example.com/other/`, no?